spec.bs

```diff
@@ -455,10 +455,10 @@ spec: html; urlPrefix: https://www.rfc-editor.org/rfc/
         - |document|'s [=Document/origin=] is an [=opaque origin=].
         - |document| is not [=allowed to use=] the <code><a href="#browsing-topics-policy-controlled-feature">browsing-topics</a></code> feature.
         - |document| is not [=allowed to use=] the <code><a href="#interest-cohort-policy-controlled-feature">interest-cohort</a></code> feature.
-        - The user preference setting disallows the access to topics from |topLevelDocument| given |document|'s [=Document/origin=].
+        - The user preference setting disallows access to topics from |topLevelDocument| given |document|'s [=Document/origin=].
```

Comment:
There's another place in the file that says "disallows the access" (I'm unable to add comment in-place). Can you fix that place as well?

Reply:
Thanks Yao — I fixed the other "disallows the access".